common/table.yaml

```
@@ -52,6 +52,7 @@ datasets:
 
 - data_type_def: VectorIndex
   data_type_inc: Index
+  dtype: int
```

You wrote in the issue that it should be unsigned int. Also, to be explicit, we should specify the minimum precision.

```suggestion
  dtype: uint8
```